src/world/area_kmr/kmr_12/events.c

```diff
@@ -188,7 +188,7 @@ StaticNpc N(goombaNpc) = {
 };
 
 NpcGroupList N(npcGroupList) = {
-    NPC_GROUP(N(goombaNpc), BATTLE_ID(1, 0, 3)),
+    NPC_GROUP(N(goombaNpc), BATTLE_ID(0, 1, 0, 3)),
```

Comment:
Why did you add `unk` to the macro if you weren't going to use it? I've never seen the most significant byte be used it battle IDs.

Reply:
It is used in area_02. https://github.com/pmret/papermario/blob/b84a86d8eb95ca9cb4a85a5b31fa631e825fa897/src/world/area_arn/arn_02/events.c#L225

I had to add it for this, and then it threw an error for goomba due to not enough args.

Reply:
Ah, I missed this... no idea what it does; guess we should keep it anyways